can/io/mf4.py

```diff
@@ -0,0 +1,160 @@
+# coding: utf-8
+
+"""
+Contains handling of MF4 logging files.
```

Comment:
Do you plan on adding a reader as well? If not, we should mention that only writing is supported as of now.

Reply:
Yes I do. The tricky part is that the mdf format makes a distinction between the normal and error frames so they are stored separately. I'll have to see how to interleave the two types for the reader part.